precovery/frame_db.py

```diff
@@ -309,28 +314,28 @@ def frames_for_bundle(
         select_stmt = (
             sq.select(self.frames)
             .where(
-                self.frames.c.mjd_mid >= bundle.start_epoch,
-                self.frames.c.mjd_mid <= bundle.end_epoch,
+                self.frames.c.exposure_mjd_mid >= bundle.start_epoch,
+                self.frames.c.exposure_mjd_mid <= bundle.end_epoch,
             )
-            .order_by(self.frames.c.mjd_mid.desc())
+            .order_by(self.frames.c.exposure_mjd_mid.desc())
         )
         rows = self.dbconn.execute(select_stmt)
         for row in rows:
             yield HealpixFrame(*row)
 
     def mjd_bounds(self) -> Tuple[float, float]:
         """
-        Returns the minimum and maximum mjd_mid of all frames in the index.
+        Returns the minimum and maximum exposure_mjd_mid of all frames in the index.
         """
         select_stmt = sq.select(
-            sqlfunc.min(self.frames.c.mjd_mid, type=sq.Float),
-            sqlfunc.max(self.frames.c.mjd_mid, type=sq.Float),
+            sqlfunc.min(self.frames.c.exposure_mjd_mid, type=sq.Float),
+            sqlfunc.max(self.frames.c.exposure_mjd_mid, type=sq.Float),
         )
         first, last = self.dbconn.execute(select_stmt).fetchone()
         return first, last
 
     def frame_bundles(
-        self, window_size_days: int, mjd_start: float, mjd_end: float
+        self, window_size_days: int, exposure_mjd_start: float, mjd_end: float
```

Comment:
I don't think renaming this one to `exposure_mjd_start` makes sense.

Reply:
Ah yes, good catch. Let me fix it and then I'll merge.